### ISSUE TITLE
[BUG] Remote VLC will not play some files

### ISSUE THREAD

**Issue by LivingWithHippos** [LivingWithHippos/unchained-android#386]
Some files are refused with a generic error "your input can't be opened: VLC is unable to open the MRL ...."

Test if it's related to spaces in the name or what else

**Comment by Mahava** [comment]
I have this issue too.  Tested with the same RD link, and it playes when I open the network tab ns paste the url in the box.  The same link won't play if I send it via Unchained

**Comment by LivingWithHippos** [comment]
> I have this issue too.  Tested with the same RD link, and it playes when I open the network tab ns paste the url in the box.  The same link won't play if I send it via Unchained

If its not a problem can you send the original link to livingwithhippos at gmail dot com?

**Comment by Mahava** [comment]
yes, Ill send it later today

**Comment by Mahava** [comment]
Sent.

**Comment by LivingWithHippos** [comment]
Yeah it's just some bug with spaces I think, I need to encode it so it uses %20 etc

```kt
val originalString = "https://.real-debrid.com/d/RANDOMW/Link With Spaces"
val url = URLEncoder.encode(originalString, "UTF-8")
println("Unencoded string: $originalString")
println("Encoded url: $url")
```

```
Unencoded string: https://.real-debrid.com/d/RANDOMW/Link With Spaces
Encoded url: https%3A%2F%2F.real-debrid.com%2Fd%2FRANDOMW%2FLink+With+Spaces
```

**Comment by LivingWithHippos** [comment]
> I have this issue too. Tested with the same RD link, and it playes when I open the network tab ns paste the url in the box. The same link won't play if I send it via Unchained

Sorry I don´t understand, which one is the working one? sending to vlc on your phone using the "send to player" button or copying the link in your pc vlc? When you copy the link to your pc, which button do you use? The copy or the share one?

**Comment by LivingWithHippos** [comment]
nevermind, I forgot about remote VLC. Fixed it, it was the spaces
